cinnamonvantage@garlayntoji/info.json

```
@@ -0,0 +1 @@
+{"author": "Garlayn Toji"}
```

```suggestion
{"author": "Garlayntoji"}
```

This should be your GitHub username.